src/add.js

```diff
@@ -0,0 +1,21 @@
+function Task(description) {
+  this.description = description;
+  this.completed = false;
+}
+
+let tasks = [
+
+];
+
+export default function addTask(description) {
+  const newTask = new Task(description);
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+  tasks.push(newTask);
+  let index = 1;
+  tasks.forEach((item) => {
+    item.index = index;
+    index += 1;
+  });
+  const toStore = JSON.stringify(tasks);
+  localStorage.setItem('tasks', toStore);
```

Comment:
- [ ] Please make sure that the functionality of the **add Task to UI** and Local-storage works perfect. As you can see from the video below, I couldn't be able to add any task since to add task functionality is not working and due to that I was not able to the check the remove functionality too. So please make sure that you fix it including the functionality of the remove task. :pray: 

[Screencast from 02-23-2023 12:49:00 AM.webm](https://user-images.githubusercontent.com/92303891/220798280-feecdf73-5c92-47ce-8d61-592f9aff354d.webm)

Reply:
Hi @Owusu-Desmond, please watch this video 
https://www.loom.com/share/43ee7ccc75984a968537a5bd0786a4fa 

I didn't change anything in my code. Can you help me? I don't know what's the problem.

Reply:
Oh! I found the problem, I miss the initialization of the 'tasks' item on the local storage.   I solved and I will request a new review. Thank you for your time

Reply:
Nice you have been able to debug @Alejandro-Bernal-M